app/models/product_importer.rb

```diff
@@ -0,0 +1,314 @@
+require 'roo'
+
+class ProductImporter
+  extend ActiveModel::Naming
+  include ActiveModel::Conversion
+  include ActiveModel::Validations
```

Comment:
There are a couple of places where they get used. I could maybe strip them out though.